integration-tests/test_satellite.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+import subprocess
+import os
+import yaml
+import traceback
+import contextlib
+from time import sleep
+import re
+
+
+pytestmark = pytest.mark.usefixtures("register_subman")
+
+
+def test_obfuscation(insights_client, tmp_path):
+    """
+    Test if the satellite hostname is obfuscated
+    Related to https://bugzilla.redhat.com/show_bug.cgi?id=1963917
+
+    """
+    # Register system to insights
+    insights_client.register()
+    # assert insights_client.is_registered
```

Comment:
hm isn't this needed?

Reply:
Running "assert insights_client.is_registered" always failed without any reason, which blocks me debugging the following code.

Reply:
it works with sleep(5)